srcpkgs/base-files/files/_vkpurge

```diff
@@ -0,0 +1,29 @@
+#compdef vkpurge
+
+_vkpurge() {
+	local line state
+
+	_arguments -C \
+		'1: :->subcmd' \
+		'*::kernel:->kernels'
+
+	case "$state" in
+	subcmd)
+		_values 'subcommand' \
+			'list[List removable kernel versions]' \
+			'rm[Remove removable kernel versions]'
+		;;
+	kernels)
+		case "$line[1]" in
```

Comment:
Would be nice to distinguish between `vkpurge list` and `vkpurge rm` under the same tag name. The doc says:

> Completion functions for commands  that  have  sub-commands  usually modify this field to contain the name of the command followed by a minus sign and the sub-command.  For example, the completion function for the `cvs` command sets this field to `cvs-add` when completing arguments to the `add` subcommand.

```suggestion
		curcontext="${curcontext%:*:*}:vkpurge-$line[1]:"
		case "$line[1]" in
```